tests/check/check_signed_video_auth.c

```diff
@@ -1091,10 +1091,10 @@ START_TEST(lost_all_nalus_between_two_seis)
     // SI                ->   (valid) -> .P
     //  IPPPSS           ->   (valid) -> .....P
     //       SI          -> (invalid) -> MMMM.P (4 missed)
-    //        IPPPSI     -> (invalid) -> N....P
+    //        IPPPSI     ->   (valid) -> U....P
```

Comment:
Why does the I-frame get an unknown validation status? That should not be possible once the stream has been detected as signed. Once a SEI has occurred, every hashable NALU should get either 'N' or '.'